.gitignore

```diff
@@ -1 +1,7 @@
-.idea/
\ No newline at end of file
+.idea/
+.DS_Store
+lint.txt
+
+companybindex/companybindex
+
+errors/
```

Comment:
Please update the `.gitignore` in line with the [styleguides](https://github.com/companieshouse/styleguides/blob/master/gitignore/go)

Reply:
.gitignore updated to conform to styleguide standards